app/templates/components/settings/email-preferences-section.hbs

```diff
@@ -43,23 +42,23 @@
           {{t 'New Paper is Submitted to your Event'}}
         </div>
         <div class="ui column eight wide right aligned">
-          {{ui-checkbox class='toggle' checked=event.newPaper onChange=(action (mut event.newPaper))}}
+          {{ui-checkbox class='toggle' checked=preference.newPaper onChange=(action 'save' 'newPaper' preference)}}
```

Comment:
Just on change `(action 'save' preference)` would do. The property would already have been modified.